userguide/sharing.rst

```diff
@@ -1637,10 +1638,10 @@ summarizes the settings that can be configured when adding a portal.
    |                       | menu      | :guilabel:`Discovery Auth Method` is set to *CHAP* or *Mutual CHAP*.        |
    |                       |           |                                                                             |
    +-----------------------+-----------+-----------------------------------------------------------------------------+
-   | IP address            | drop-down | Select the IPv4 or IPv6 address associated with an interface or the         |
-   |                       | menu      | wildcard address of *0.0.0.0* (any interface). Click :guilabel:`ADD` to add |
-   |                       |           | another IP address to the portal. Click the associated :guilabel:`DELETE`   |
-   |                       |           | button to remove the extra IP address.                                      |
+   | IP address            | drop-down | Select the IPv4 or IPv6 addresses associated with an interface or the       |
+   |                       | menu      | wildcard addresses :literal:`0.0.0.0` (all IPv4 addresses) and/or           |
+   |                       |           | :literal:`::` (all IPv6 addresses). To add more IP addresses on another     |
+   |                       |           | :guilabel:`Port`, click :guilabel:`ADD`.                                    |
```

Comment:
I think there is also an value for "all addresses", although I'm not sure what was chosen.  If any of the "all" values are picked, it disables choosing other individual addresses, so the last sentence ("To add more") should become a second sentence, and the "all" aliases mentioned in a third sentence.

Reply:
Just rechecked, there is no "all addresses" option, just the 0.0.0.0 and :: wildcards.